pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -108,28 +89,15 @@ func WithMinBackoffSeconds(minBackoffSeconds int32) Option {
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=workloads/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=admissionchecks,verbs=get;list;watch
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=provisioningrequestconfigs,verbs=get;list;watch
-
```

Comment:
Ah, this is likely why you lost the perimissions from manifests, I'm not sure why but kubebuilder requires this holy empty line:)

Reply:
Wow, thank you